backend/endpoints/rom.py

```diff
@@ -10,47 +13,115 @@
 router = APIRouter()
 
 
+class RomSchema(BaseModel):
```

Comment:
Is there a way to `merge` somehow this `RomSchema` with the `Rom` model? I guess as all the `Rom` attributes are `Columns` it can't be translated directly to python types, but I was just wondering if we can just have a single place where to put the models schamas and extend from there.